src/Stripe.net/Services/ApplePayDomains/ApplePayDomainService.cs

```diff
@@ -21,44 +21,53 @@ public ApplePayDomainService(string apiKey)
         {
         }
 
+        public override string ObjectName => "apple_pay_domain";
+
         public virtual ApplePayDomain Create(ApplePayDomainCreateOptions options, RequestOptions requestOptions = null)
         {
-            return this.Post($"{Urls.BaseUrl}/apple_pay/domains", requestOptions, options);
+            return this.CreateEntity(options, requestOptions);
         }
 
         public virtual Task<ApplePayDomain> CreateAsync(ApplePayDomainCreateOptions options, RequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
         {
-            return this.PostAsync($"{Urls.BaseUrl}/apple_pay/domains", requestOptions, cancellationToken, options);
+            return this.CreateEntityAsync(options, requestOptions, cancellationToken);
         }
 
         public virtual ApplePayDomain Delete(string domainId, RequestOptions requestOptions = null)
         {
-            return this.DeleteEntity($"{Urls.BaseUrl}/apple_pay/domains/{domainId}", requestOptions);
+            return this.DeleteEntityNew(domainId, null, requestOptions);
```

Comment:
I'm assuming there's no way to get rid of the `null` for most DELETE?

Reply:
No, this is possible too. I added an override to be able to call `DeleteEntityNew` without an options class.